robottelo/api/utils.py

```diff
@@ -771,6 +771,35 @@ def wait_for_syncplan_tasks(repo_backend_id=None, timeout=10, repo_name=None):
         time.sleep(2)
 
 
+def wait_for_errata_applicability_task(host_id, search_rate=1, max_tries=10, poll_rate=None,
+                                       poll_timeout=15):
+    """Search the generate applicability task for given host and make sure it finishes
```

Comment:
```suggestion
    """Search the generate applicability task for given host and return relevant task
```
Above looks more relevant as helper does not wait to finish task, right ?

Reply:
It pools the task, so it waits for it to finish here:
```
task.poll(poll_rate=poll_rate, timeout=poll_timeout)
```